src/stirling/source_connectors/socket_tracer/socket_trace_connector.h

```diff
@@ -341,6 +343,7 @@ class SocketTraceConnector : public SourceConnector, public bpf_tools::BCCWrappe
   FRIEND_TEST(SocketTraceConnectorTest, ConnectionCleanupInactiveDead);
   FRIEND_TEST(SocketTraceConnectorTest, ConnectionCleanupInactiveAlive);
   FRIEND_TEST(SocketTraceConnectorTest, ConnectionCleanupNoProtocol);
+  FRIEND_TEST(SocketTraceConnectorTest, MuxBasic);
```

Comment:
I think we can remove this now.

Reply:
Oops, this was leftover from before. Removed in the latest revision.